### PR TITLE
TEDU-54 Reorganized Campaigns: Fixed broken table and changed title

### PR DESCRIPTION
**Pull request by swati-thacker13**

### CODE REVIEW

File: pages/06.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md [hunk 1]
```diff
@@ -38,16 +38,13 @@ The actions that Mautic offers in a Campaign include:
...
-|**Send email to user**| Sends an Email to an entity other than the contact. This may be a Mautic user, the Contact’s owner, or non-users. 
-
->>> Emails sent using this action do not generate any statistics for Contacts or Emails.|
+|**Send email to user**| Sends an Email to an entity other than the contact. This may be a Mautic user, the Contact’s owner, or non-users. Emails sent using this action do not generate any statistics for Contacts or Emails.|
 |**Send marketing message**| Sends a message using the Contact's preferred Channel.|
 |**Update contact**| Updates the existing Contact's fields with the specified values.|
 |**Update contact's primary company**| Updates the existing Contact's primary Company fields with the specified value. See [documentation on Companies][companies].|
 |**Update contact owner**| Updates the Contact's owner.|
 |
```
Comment (RCheesley) [inline pages/06.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md:46]: When using the view file feature on GH I see an extra pipe here:

screenshot-github com-2021 03 02-19_16_14


```suggestion
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
One surplus pipe otherwise all good!

File: pages/06.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md (hunk 1 above)
Comment (swati-thacker13) [reply pages/06.Campaigns/02. Using Campaign Builder/01.Actions/docs.en.md]: This pipe was used to close the table. I removed the pipe. Can you please check if there's a closing line in the table?

**Comment by RCheesley** [comment]
Sure, for info you can preview the file in its changed state using this menu item:

screenshot-github com-2021 03 03-10_10_25

It looks OK now, hopefully it will work OK on Grav too!

**Review by RCheesley — APPROVED** [review]

**Pull request merged**